<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QC2Widget</class>
 <widget class="QWidget" name="QC2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input</string>
  </property>
  <widget class="QPushButton" name="button_proceed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>299</width>
     <height>393</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Foil Type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="button_GE21_M2">
         <property name="text">
          <string>GE2/1 M2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="button_GE21_M3">
         <property name="text">
          <string>GE2/1 M3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="button_GE21_M6">
         <property name="text">
          <string>GE2/1 M6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="button_GE21_M7">
         <property name="text">
          <string>GE2/1 M7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="button_ME0">
         <property name="text">
          <string>ME0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="button_10_10">
         <property name="text">
          <string>10*10</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_process">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>QC_Long</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Preparation_QC_Long</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Relative Humidity (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_rh"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Channel Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_channel_number">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tester's Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_tester"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Batch Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_batch_number"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Foil Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_foil_number"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Temperature (&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_temperature"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>601</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
